Document/MediaManager.php

```diff
@@ -30,11 +30,13 @@ public function save($entity, $andFlush = true)
     {
         // BC compatibility for $context parameter
         if ($andFlush && is_string($andFlush)) {
+            trigger_error('Since version 2.2.7, passing context in argument 2 for '.__METHOD__.'() is deprecated and support for it will be removed in 3.0. Use '.$this->class.'::setContext() instead.', E_USER_DEPRECATED);
```

Comment:
Check also this [Symfony's deprecation message](https://github.com/symfony/symfony/blob/ae470cdf148d5b779623f6922e45160fe14e9b6f/src/Symfony/Bundle/FrameworkBundle/Templating/Helper/RequestHelper.php#L38), it's practically identical.